rhine/rhine.cabal

```
@@ -129,16 +129,15 @@ library
 
   -- Other library packages from which modules are imported.
   build-depends:
-                     , dunai        ^>= 0.9
+                     , dunai        >= 0.11
```

```suggestion
                     , dunai        ^>= 0.11
```
I think it's better to have a strict bound, since changes in `dunai` typically break `rhine`.